steepshot_io/graph/views.py

```diff
@@ -3,7 +3,8 @@
 from django.shortcuts import render
 from django.views.generic import View
 from django.conf import settings
-
+from json.decoder import JSONDecodeError
+from requests.exceptions import HTTPError
```

Comment:
Add import of 'ConnectionError' from requests module. ConnectionError which will raise from requests module will not handling.